ports/arrow/brotli.patch

```
@@ -0,0 +1,28 @@
+diff --git a/cpp/cmake_modules/FindBrotli.cmake b/cpp/cmake_modules/FindBrotli.cmake
+index 4053b79ca7..bdca85fab4 100644
+--- a/cpp/cmake_modules/FindBrotli.cmake
++++ b/cpp/cmake_modules/FindBrotli.cmake
+@@ -21,6 +21,23 @@ if(Brotli_FOUND)
+   return()
+ endif()
+ 
++if(ARROW_PACAKGE_KIND STREQUAL "vcpkg")
++  set(find_package_args "")
++  if(Brotli_FIND_VERSION)
++    list(APPEND find_package_args ${Brotli_FIND_VERSION})
++  endif()
++  if(Brotli_FIND_QUIETLY)
++    list(APPEND find_package_args QUIET)
++  endif()
```

Next time, would you mind making this REQUIRED? :) 
Other than that, this looks good to me!
Thank you!

---

This spot is in a find module forwarding to vcpkg's unofficial config. IMO `REQUIRED` is only possible if the original call was `REQUIRED` and the absence of vcpkg config shall raise an immediate error despite other search options.

---

OK. I'll add the following:

```cmake
  if(Brotli_FIND_REQUIRED)
    list(APPEND find_package_args REQUIRED)
  endif()
```